arch/arm/src/sama5/sam_tc.c

```diff
@@ -1443,19 +1445,19 @@ int sam_tc_divisor(uint32_t frequency, uint32_t *div, uint32_t *tcclks)
   if (div)
     {
       uint32_t value = sam_tc_freqdiv_lookup(ftcin, ndx);
-      tmrinfo("return div=%lu\n", (unsigned long)value);
+      tmrinfo("return div=%lu\n", (unsigned long) value);
       *div = value;
     }
 
   /* Return the TCCLKS selection */
 
   if (tcclks)
     {
-      tmrinfo("return tcclks=%08lx\n", (unsigned long)TC_CMR_TCCLKS(ndx));
+      tmrinfo("return tcclks=%08lx\n", (unsigned long) TC_CMR_TCCLKS(ndx));
```

Comment:
```suggestion
      tmrinfo("return tcclks=%08lx\n", (unsigned long)TC_CMR_TCCLKS(ndx));
```

Reply:
I put the space in because that was the only way checkpatch.sh was happy and certainly not what i would usually do! I will try again tomorrow as I think the script was having a bad day (which isn't even possible).